src/common/footer/ExtendedFooter.jsx

```diff
@@ -39,10 +36,10 @@ const ExtendedFooter = () => {
       title: 'Blog Page'
     },
     {
-      name: 'Meetup',
-      url: 'https://www.meetup.com/reactplay-bengaluru/',
-      icon: <FaMeetup className="icon" />,
-      title: 'Meetup Page'
+      name: 'Email',
+      url: 'mailto:contact@reactplay.io',
+      icon: <FaEnvelope className="icon" />,
+      title: 'Email'
```

Comment:
please try to add an another object for `email` we added a `meetup` link

Reply:
done